docs/jwst/extract_1d/description.rst

```diff
@@ -450,6 +450,38 @@ where `flux` is the extracted spectral data, and the data are from channel 4 for
 
 Extraction for NIRISS SOSS Data
 -------------------------------
-For NIRISS SOSS data, the two spectral orders overlap slightly, so a specialized extraction
-algorithm known as ATOCA (Algorithm to Treat Order ContAmination) is used...
-Link paper 
+For NIRISS SOSS data, spectral orders 1 and 2 overlap slightly at longer wavelengths so a specialized extraction
+algorithm known as ATOCA (Algorithm to Treat Order ContAmination, Darveau-Bernier et al., 2022) is made available in
+the ``extract_1d`` step. This routine constructs a linear model of each pixel on the detector and treats the underlying
+incident spectrum as a free variable to simultaneously extract the cross-contaminated spectra, which are accurate to
+10ppm over the full spectral range.
```

Comment:
it's a bit unclear to me what the accuracy of 10 ppm refers to - is this the uncertainty in the input data, the tolerance that ATOCA was designed to, the total uncertainty in the decontaminated spectra, the uncertainty added by ATOCA...?